boards/opentitan/earlgrey-cw310/Makefile

```diff
@@ -40,23 +40,38 @@ qemu-app-gdb : $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
 	$(QEMU) -s -S -M opentitan -kernel $^ -bios $(OPENTITAN_BOOT_ROM) -device loader,file=$(APP),addr=0x20030000 -nographic -serial mon:stdio
 
 flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
-	$(OPENTITAN_TREE)/util/fpga/cw310_loader.py --firmware $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
+	$(call check_defined, OPENTITAN_TREE)
+	$(OPENTITAN_TREE)/bazelisk.sh run //sw/host/opentitantool -- \
```

Comment:
I would prefer not to call bazel here. I just tested this and bazel first had to download and install something, it makes the entire process clunky